release/_bom.sh

```diff
@@ -67,15 +67,45 @@ function generate_api_jars {
 		rm -f "${name}-${version}.jar"
 	done
 
-	for portal_jar in portal-impl portal-kernel support-tomcat util-bridges util-java util-slf4j util-taglib
-	do
-		_manage_bom_jar "${_BUNDLES_DIR}/tomcat/webapps/ROOT/WEB-INF/shielded-container-lib/${portal_jar}.jar"
-	done
+	if [[ "${_PRODUCT_VERSION}" == "7.3."* ]]
+	then
+		_manage_bom_jar "${_BUNDLES_DIR}/tomcat-9.0.83/lib/ext/portal-kernel.jar"
```

Comment:
@lucasmiranda0, the tomcat version shouldn't be hard coded. Can you please test your fix running a `build_release` locally?

Thanks